iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/InjectionObservabilityImpl.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network.impl.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.Injection;
+import com.powsybl.iidm.network.extensions.InjectionObservability;
+import org.jgrapht.alg.util.Pair;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public class InjectionObservabilityImpl<T extends Injection<T>> extends AbstractExtension<T>
+        implements InjectionObservability<T> {
+
+    private boolean observable;
+
+    private final Pair<Float, Boolean> standardDeviationP;
+
+    private final Pair<Float, Boolean> standardDeviationQ;
+
+    private final Pair<Float, Boolean> standardDeviationV;
+
+    public InjectionObservabilityImpl(T component, boolean observable,
+                                      float standardDeviationP, boolean redundantP,
+                                      float standardDeviationQ, boolean redundantQ,
+                                      float standardDeviationV, boolean redundantV) {
+        super(component);
+        this.observable = observable;
+        this.standardDeviationP = new Pair<>(standardDeviationP, redundantP);
+        this.standardDeviationQ = new Pair<>(standardDeviationQ, redundantQ);
+        this.standardDeviationV = new Pair<>(standardDeviationV, redundantV);
+    }
+
+    public boolean isObservable() {
+        return observable;
+    }
+
+    @Override
+    public InjectionObservability<T> setObservable(boolean observable) {
+        this.observable = observable;
+        return this;
+    }
+
+    @Override
+    public float getStandardDeviationP() {
+        return standardDeviationP.getFirst();
+    }
+
+    @Override
+    public InjectionObservabilityImpl<T> setStandardDeviationP(float standardDeviationP) {
+        this.standardDeviationP.setFirst(standardDeviationP);
+        return this;
+    }
+
+    @Override
+    public boolean isRedundantP() {
+        return standardDeviationP.getSecond();
```

Comment:
@sylvlecl Is it possible to have standard deviation but no redundancy information?
If so, maybe return `false` by default (or the getters return `Boolean` that can be null?)
Same remark for other redundancy getters and observability getters.

Reply:
I must say I don't know if this "redundancy" information is "standard" in all state estimators ...
Same for standard deviation.

It's likely that some implementations will not bother to compute them (think for instance a machine learning based state estimator). So we should probably make them nullable indeed ...